tests/device_tests/test_multisig_change.py

```diff
@@ -84,70 +86,70 @@ class TestMultisigChange:
     #   tx: b0946dc27ba308a749b11afecc2018980af18f79e89ad6b080b58220d856f739
     #   input 1: 0.555 BTC
 
-    multisig_in1 = proto.MultisigRedeemScriptType(
+    multisig_in1 = messages.MultisigRedeemScriptType(
         nodes=[node_ext2, node_ext1, node_int],
         address_n=[0, 0],
         signatures=[b"", b"", b""],
         m=2,
     )
 
-    multisig_in2 = proto.MultisigRedeemScriptType(
+    multisig_in2 = messages.MultisigRedeemScriptType(
         nodes=[node_ext1, node_ext2, node_int],
         address_n=[0, 1],
         signatures=[b"", b"", b""],
         m=2,
     )
 
-    multisig_in3 = proto.MultisigRedeemScriptType(
+    multisig_in3 = messages.MultisigRedeemScriptType(
         nodes=[node_ext1, node_ext3, node_int],
         address_n=[0, 1],
         signatures=[b"", b"", b""],
         m=2,
     )
 
     # 2N9W4z9AhAPaHghtqVQPbaTAGHdbrhKeBQw
-    inp1 = proto.TxInputType(
+    inp1 = messages.TxInputType(
         address_n=[H_(45), 0, 0, 0],
         amount=50000000,
         prev_hash=TXHASH_16c6c8,
         prev_index=1,
-        script_type=proto.InputScriptType.SPENDMULTISIG,
+        script_type=messages.InputScriptType.SPENDMULTISIG,
         multisig=multisig_in1,
     )
 
     # 2NDBG6QXQLtnQ3jRGkrqo53BiCeXfQXLdj4
-    inp2 = proto.TxInputType(
+    inp2 = messages.TxInputType(
         address_n=[H_(45), 0, 0, 1],
         amount=34500000,
         prev_hash=TXHASH_d80c34,
         prev_index=0,
-        script_type=proto.InputScriptType.SPENDMULTISIG,
+        script_type=messages.InputScriptType.SPENDMULTISIG,
         multisig=multisig_in2,
     )
 
     # 2MvwPWfp2XPU3S1cMwgEMKBPUw38VP5SBE4
-    inp3 = proto.TxInputType(
+    inp3 = messages.TxInputType(
         address_n=[H_(45), 0, 0, 1],
         amount=55500000,
         prev_hash=TXHASH_b0946d,
         prev_index=0,
-        script_type=proto.InputScriptType.SPENDMULTISIG,
+        script_type=messages.InputScriptType.SPENDMULTISIG,
         multisig=multisig_in3,
     )
 
-    def _responses(self, inp1, inp2, change=0):
+    def _responses(inp1, inp2, change=0):
```

Comment:
this ... looks like it works, but it's wrong. the function should be declared a `@staticmethod`, otherwise the first argument only works by coincidence.

anyway please remove the whole class and instead move the elements to top level (inputs should probably be named in uppercase, `_responses` can stay as is)

Reply:
Class removed and global variables capitalized in 946823a